ipaserver/install/adtrust.py

```diff
@@ -344,13 +345,65 @@ def add_new_adtrust_agents(api, options):
     if new_agents:
         add_hosts_to_adtrust_agents(api, new_agents)
 
-        print("""
+        kwargs = {u'version': u'2.236',
+                  u'enable_compat': options.enable_compat}
```

Comment:
Should you add a comment that this particular version is being used because this is the version the command was added in? It was obvious once I looked at API.txt but I had the whole diff in front of me. I wonder if this will become some magic value in the future?